src/plugins/kibana_utils/docs/state_containers/react.md

```diff
@@ -9,7 +9,7 @@
 ```ts
 import { createStateContainer, createStateContainerReactHelpers } from 'src/plugins/kibana_utils';
 
-const container = createStateContainer({}, {});
+const container = createStateContainer({});
```

Comment:
It seemed to be reasonable to also make transitions optional. Just like selectors